### DISCUSSION TITLE
[macOS] Issues starting application that requires GPU remotely

### DISCUSSION THREAD

**Discussion by guanzhangrtk** [FreeRDP/FreeRDP#7114] [Q&A]
Hi, I'm on a Mac trying to connect to a Windows 10 computer via `xfreerdp`. When I try to remotely start some application that requires GPU, they will fail to start because it sees the "Microsoft Remote Display Adapter" instead of the GPU on the remote system.

However, if I physically start the application on the Windows 10 machine and then connect to it via `xfreerdp`, everything works fine. So is there some option to tell xfreerdp to use the remote host's GPU instead of the remote display adapter?

I did some basic Googling but couldn't find an answer, I also tried enabling "Use hardware graphics adapters for all Remote Desktop Services sessions" on the Windows 10 host to no avail. Sorry in advance if this is something obvious, thanks!

**Comment by guanzhangrtk** [comment]
Found this after some more Googling: https://stackoverflow.com/questions/51705471/current-state-and-solutions-for-opengl-over-windows-remote

Creating the batch file to disconnect the rdp session, start the OpenGL program then re-connecting to the host works, but would be nice if there is a better solution.

**Reply by akallabeth** [reply]
nothing `FreeRDP` can do here, this is a server side setting, nothing the client can influence.

**Reply by guanzhangrtk** [reply]
Are you aware of what settings are available for Windows 10 Professional? As mentioned the setting that is supposed to work doesn't work for me, so not sure whether it's possible to set this up or not, thanks!
